website/package.json

```diff
@@ -41,6 +29,76 @@
     "styled-components": "^5.3.5",
     "typescript": "^4.6.3"
   },
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build --out-dir build/open-source/spectacle",
+    "clear": "docusaurus clear",
+    "serve": "formideploy serve",
+    "deploy:stage": "formideploy deploy --staging",
+    "deploy:prod": "formideploy deploy --production",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "typecheck": "tsc",
```

Comment:
It's probably worth moving this to a wireit script as well, and renaming to `"types:check"` just for consistency with the other packages.

Reply:
Agreed. And we get caching as well.

Reply:
I forgot about https://github.com/FormidableLabs/spectacle/issues/1201

Maybe for now, we just remove this all together?

Reply:
I'm gonna vote to keep this as is for now; and then in a new PR it can be rewritten to resolve https://github.com/FormidableLabs/spectacle/issues/1201 and the `types:check` task from https://github.com/FormidableLabs/spectacle/issues/1203 too